bolt-modules/boltlib/spec/fixtures/modules/test/tasks/undef.sh

```diff
@@ -0,0 +1 @@
+echo -n "$PT_undef"
```

Comment:
Since this doesn't actually work (the parameter name is wrong), I think we can either use an existing task instead, or should fix this and verify that the output is correct.